docs/src/websocket_doc.md

```diff
@@ -0,0 +1,237 @@
+
+# WebSocket API Documentation
+
+## Table of Contents
+- [getChats](#getChats)
+- [createEmptyChat](#createEmptyChat)
+- [joinChat](#joinChat)
+- [getChatDetails](#getChatDetails)
+- [newMessage](#newMessage)
+- [getChatMessages](#getChatMessages)
+
+---
+
+## getChats
+
+### Description:
+Retrieve all chats in which the user participates.
+
+### Request:
+
+```json
+{
+  "type": "getChats",
+  "data": {}
+}
+```
+
+### Response:
+
+```json
+{
+  "type": "chatsList",
+  "data": [
+    {
+      "id": "int",                // chat ID
+      "participants": [
+        {
+          "id": "int",            // participant ID
+          "username": "string"    // Member user name
+        }
+      ],
+      "created_at": "string",     // Date and time of chat creation
+      "updated_at": "string"      // Date and time of the last chat update
+    }
+  ]
+}
+```
+
+---
+
+## createEmptyChat
+
+### Description:
+Create an empty chat entity, with no participants. The server creates a new chat and returns its identifier, as well as the timestamps of creation and last update.
+
+### Request:
+
+```json
+{
+  "type": "createEmptyChat",
+  "data": {
+    "name": "string"  // Chat Name (optional)
```

Comment:
Our chat doesn't have a name yet, so I think it should be removed with the documentation.